docs/src/main/asciidoc/config-reference.adoc

```diff
@@ -359,7 +378,8 @@ Properties in the profile aware file have priority over profile aware properties
 
 [WARNING]
 ====
-The profile aware file must be present in the exact same location as the main `application.properties` file.
+A profile aware file is only loaded if the unprofiled `application.properties` is also available in the same location.
```

Comment:
Wondering if we should add ``(respectively, `.yml` or `.yaml`)`` here? Not sure if it makes sense. I know we avoid having YAML content in this doc but I'm wondering if we should be clear that the extension needs to be exactly the same.

Reply:
We can add that. It should be ok.